Makefile

```diff
@@ -199,7 +200,8 @@ magic-folder:
 	python -m pip install -r requirements/pyinstaller.txt && \
 	cp misc/magic-folder/* build/magic-folder && \
 	pushd build/magic-folder && \
-	python -m pip install . && \
+	python $(SCRIPTS)/reproducible-pip.py install --require-hashes -r requirements/base.txt && \
```

Comment:
Ironically, this line is _breaking_ reproducibility on macOS (as per [this test run](https://github.com/gridsync/gridsync/pull/355/checks?check_run_id=3116409975)), due, apparently, to the pyrsistent 0.16.1 wheel not building in a deterministic fashion... Can you try pip installing the requirements with `CFLAGS=-g0` and see if doing so will result in a pass? (I had to do this previously in order to build reproducible tahoe binaries; see, e.g., the `frozen-tahoe` make target in the same Makefile).